qa/workunits/rados/test_rados_tool.sh

```diff
@@ -52,18 +52,30 @@ run() {
 DNAME="`dirname $0`"
 DNAME="`readlink -f $DNAME`"
 RADOS_TOOL="`readlink -f \"$DNAME/../rados\"`"
-if ! test -f $RADOS_TOOL ; then
-    RADOS_TOOL=$(which rados)
+if [ ! -f "$RADOS_TOOL" ] ; then
```

Comment:
@tchaikov 
Does this mean that CEPH_TMP, or something like that is also set?
Would be nice to dump all the TEMP stuff in, instead of /tmp.